cmd/antrea-agent/options.go

```diff
@@ -86,6 +86,12 @@ func (o *Options) validate(args []string) error {
 	if err != nil {
 		return fmt.Errorf("service CIDR %s is invalid", o.config.ServiceCIDR)
 	}
+	if o.config.ServiceCIDRv6 != "" {
+		_, _, err := net.ParseCIDR(o.config.ServiceCIDRv6)
+		if err != nil {
+			return fmt.Errorf("service CIDR v6 %s is invalid", o.config.ServiceCIDRv6)
```

Comment:
Service IPv6 CIDR or IPv6 Service CIDR

service -> Service (as Service is a K8s resource type).
 know it is not your code.